static/js/modules/tables.js

```diff
@@ -356,49 +422,134 @@ function initTable($table, $form, baseUrl, columns, callbacks, opts) {
-            handleResponse: handleResponseSeek,
-            afterRender: modalAfterRender.bind(undefined, donationTemplate)
-          },
-          {
-            order: [[5, 'desc']],
-            pagingType: 'simple'
-          }
-        );
-        break;
-      case 'expenditure':
-        initTable(
-          $table,
-          $form,
-          'schedules/schedule_b',
-          expenditureColumns,
-          {
-            mapQuery: mapQuerySeek,
-            handleResponse: handleResponseSeek,
-            afterRender: modalAfterRender.bind(undefined, expenditureTemplate)
-          },
-          {
-            order: [[4, 'desc']],
-            pagingType: 'simple'
+
+    var offsetCallbacks = {
+      mapQuery: mapQueryOffset
+    };
+    var seekCallbacks = {
+      mapQuery: mapQuerySeek,
+      handleResponse: handleResponseSeek
+    };
+    $tables.each(function(index, elm) {
+      var $table = $(elm);
+      var committeeId = $table.attr('data-committee');
+      var cycle = $table.attr('data-cycle');
+      var year = $table.attr('data-year');
+      var path, query;
+      switch ($table.attr('data-type')) {
+        case 'candidate':
+          initTable($table, $form, 'candidates', {}, candidateColumns, offsetCallbacks, {useFilters: true});
+          break;
+        case 'committee':
+          initTable($table, $form, 'committees', {}, committeeColumns, offsetCallbacks, {useFilters: true});
+          break;
+        case 'donation':
+          initTable(
+            $table,
+            $form,
+            'schedules/schedule_a',
+            {},
+            donationColumns,
+            {
+              mapQuery: mapQuerySeek,
+              handleResponse: handleResponseSeek,
+              afterRender: modalAfterRender.bind(undefined, donationTemplate)
+            },
+            {
+              order: [[5, 'desc']],
+              pagingType: 'simple'
+            }
+          );
+          break;
+        case 'expenditure':
+          initTable(
+            $table,
+            $form,
+            'schedules/schedule_b',
+            {},
+            expenditureColumns,
+            {
+              mapQuery: mapQuerySeek,
+              handleResponse: handleResponseSeek,
+              afterRender: modalAfterRender.bind(undefined, expenditureTemplate)
+            },
+            {
+              order: [[4, 'desc']],
+              pagingType: 'simple'
+            }
+          );
+          break;
+        case 'committee-contributor':
+          path = ['committee', committeeId, 'schedules', 'schedule_a', 'by_contributor'].join('/');
+          query = {};
+          if (year) {
+            query.year = year;
+          } else {
+            query.cycle = cycle;
           }
-        );
-        break;
-    }
+          initTable($table, $form, path, query, committeeContributorColumns, offsetCallbacks, {
+            dom: '<"results-info meta-box results-info--top"lfrip>t',
```

Comment:
No, because the DataTables API confusingly treats anything in quotes as a class in this context: https://datatables.net/reference/option/dom. Unless you prepend a hash, in which case it's an ID.